pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/metadata/SegmentMetadataImpl.java

```diff
@@ -209,7 +216,8 @@ private void loadCreationMeta(InputStream crcFileInputStream)
     }
   }
 
-  private void init(PropertiesConfiguration segmentMetadataPropertiesConfiguration) {
+  private void init(PropertiesConfiguration segmentMetadataPropertiesConfiguration,
+      PropertiesConfiguration segmentIndexPropertiesConfiguration) {
```

Comment:
no implemented yet: still need to parse this index PropertiesConfiguration but it should have everything